
my_vector: [5 , 6, 7 , 9]

other_vector:
  - -5
  - 4
  - 3

my_struct: 
  { 
    name: "Slifer", 
    passcode: 6463, 
    nums:
      [
        [1, 5, 8],
        [1, 5, 8],
        [1, 5, 8]
      ]
  }


arm_vec: [-9, -2.3, -5, 4.7]
arm_row_vec: [0.5, 6.7, -9.4, 2.2, 1.3]

arma_mat:
  [
    [1, 2, 3, 4, 5],
    [6, 7, 8, 9, 10],
    [11, 12, 13, 14, 15]
  ]